docs/CHANGELOG.md

```diff
@@ -1,5 +1,28 @@
 ### vNEXT 
 
+**Breaking changes:**
+
+- Support querying the same fields and relations with different arguments through different aliases (fixes [#126](https://github.com/join-monster/join-monster/issues/126))
+  - It is no longer guaranteed that a field's value is available under `source[fieldName]` in a custom resolver. Instead, custom resolvers on non-trivial fields should use GraphQL's default resolver to get the raw value:
+
+```javascript
+import { defaultFieldResolver } from 'graphql'
+
+const User = new GraphQLObjectType({
+  //...
+  fields: () => ({
+    //...
+    following: {
+      // ...
+      resolve: (source, args, context, info) => {
+        const rawValue = defaultFieldResolver(source, args, context, info)
```

Comment:
assuming this is only required for "non-trivial" field usage. Please confirm.

Reply:
Yes, `defaultFieldResolver` is only required for non-trivial fields.

The problem is that GraphQL only uses its `defaultFieldResolver` if you do not specify a custom `resolve` function. As soon as you do, you need to handle this yourself.

I extended the section on custom resolvers in the docs and added a link to it from the changelog, please let me know if this covers it better now 😅